File: Data Structures & Algorithms/two-integer-sum/submission-1.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        int complement;
        int i,j;
        for(i=0;i<nums.size();i++){
            complement=target-nums[i];
            for(j=i+1;j<nums.size();j++){
                if (nums[j]==complement){
                    return {i,j};
                }
            }
        }
        return {};
    }
};

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.py
class Solution:
    def longestConsecutive(self, nums):
        num_set = set(nums)
        longest = 0
        for n in num_set:
          if n - 1 not in num_set:         # only start from a sequence's beginning
            length = 1
            while n + length in num_set:  # walk forward via O(1) lookups
                length += 1
            longest = max(longest, length)
        return longest
        